src/ieee.rs

```diff
@@ -8,6 +10,12 @@ use core::marker::PhantomData;
 use core::mem;
 use core::ops::Neg;
 
+/// A floating point number that uses IEEE semantics.
+///
+/// Usually you will want the type aliases ([`Single`], [`Double`], ...) rather than using this
+/// directly.
```

Comment:
```suggestion
/// Usually you will want to use the available type aliases of this type
/// (e.g., [`Single`], [`Double`]) rather than referencing it directly.
```